app/src/main/java/org/fossasia/openevent/general/similarEvents/EventTopicsDao.kt

```diff
@@ -0,0 +1,18 @@
+package org.fossasia.openevent.general.similarEvents
+
+import android.arch.persistence.room.Dao
+import android.arch.persistence.room.Insert
+import android.arch.persistence.room.OnConflictStrategy
+import android.arch.persistence.room.Query
+import io.reactivex.Flowable
+import org.fossasia.openevent.general.event.Event
+
+@Dao
+interface EventTopicsDao {
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    fun insertSimilarEvents(events: List<Event>)
+
+    @Query("DELETE FROM Event")
+    fun deleteAllEvents()
+
+}
```

Comment:
No need of this file

Reply:
No need of this file

Reply:
Won't we require this in future to show users a list of event topics ?